pkg/scaffold/v2/makefile.go

```diff
@@ -98,7 +98,7 @@ generate: controller-gen
 docker-build: test
 	docker build . -t ${IMG}
 	@echo "updating kustomize image patch file for manager resource"
```

Comment:
The following 2 lines should be in the `deploy` target.

Reply:
moved them as suggested. thx